data/hai/hai reveal 5 philosophers.txt

```diff
@@ -964,7 +964,7 @@ mission "Hai Reveal [B11] Moving Together"
 			action
 				set "Hai Reveal [B11] Moving Together: label secret"
 			`	Remington explains. "This operation has gone on long enough that the popular opinion is that the Hai must obviously exist, even though there are no explicit voices saying as such. It has in fact almost become a joke; the perception being that, 'The Navy are blockading the Hai to hide humanity from them, because the blockade isn't doing a good job of being inconspicuous.' There have been memes of the Navy hiding their ships behind far too narrow light poles while they distract a squirrel with their headlights. Now it's a matter of being able to present good news. Place a successful spin on things."`
-			`	"Our problem," says Khaine. "Is Elenchus."`
+			`	"Our problem," says Kaine. "Is Elenchus."`
```

Comment:
Should the second part not follow on from the previous part as a single sentence, so a comma there instead of a full stop and no capital `is`?

Reply:
That is how I was taught grammar, but I keep being corrected cause apparently it's not standard for US?
That's an EJo/Road/Wiz question at the end of the day.

Reply:
There shouldn't be a full stop there. Kaine isn't actually saying it like "Our problem. Is Elenchus." I'd imagine.

Not sure why it's different across the ditch : )

Reply:
Warp's got it right, yea.